home.css

```diff
@@ -0,0 +1,43 @@
+body {
+    /* uiGradient - Visions of grandeur */
+    background: #000046;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #000046, #1CB5E0) fixed;  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #000046, #1CB5E0) fixed; /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+    /*Have to put 'fixed' on end of linear-gradient for it to work / fill entire screen*/
+}
+
+.ie {
+    /* uiGradient - Telegram */
+    background: #1c92d2;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #f2fcfe, #1c92d2) fixed;  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #f2fcfe, #1c92d2) fixed; /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+.name {
+    margin-top: 300px;
+    text-align: center;
+    font-size: 3rem;
+}
+
+.title {
+    margin-top: 5px;
+    text-align: center;
```

Comment:
same as .name class - put text align on parent so only have to declare it once

Reply:
done